run_tests.sh

```diff
@@ -1,77 +1,7 @@
 #!/usr/bin/env bash
```

Comment:
I don't think you need this file now, as running tests is only one command. But that's not a big deal anyway. Keep it if you want.

Reply:
I am merging the whole blueprint to crypto3 after and it will go anyway